mod/dfrn_request.php

```diff
@@ -514,6 +514,11 @@ function dfrn_request_post(App $a) {
 					return; // NOTREACHED
 				}
 
+				if (blocked_url($url)) {
+					notice( t('Blocked domain') . EOL);
```

Comment:
I think Settings.md, as there is all the admin panel documentation, and maybe referencing it from somewhere else.